CONTRIBUTING.md

```diff
@@ -60,6 +60,13 @@ cargo run -- venv
 cargo run -- pip install requests
 ```
 
+### Testing on Windows
+
+When testing debug builds on Windows, the stack can overflow resulting in a `STATUS_STACK_OVERFLOW` error code.
+This is due to a small stack size limit on Windows that we encounter when running unoptimized builds — the release
+builds do not have this problem. We [added a `UV_STACK_SIZE` variable](https://github.com/astral-sh/uv/pull/941) to
+bypass this problem during testing. We recommend bumping the stack size from the default of 1MB to 2MB, e.g., `UV_STACK_SIZE=2000000`.
```

Comment:
Should we move this into a code block with proper powershell export syntax?

Reply:
Sure, makes sense.

Reply:
fwiw I'll need to Google this. If someone else knows it off the top of there head and gets to it before me feel free to suggest / commit it :)